Web view/Web app/src/Institution VIew/AddLecturer/addLecturer.js

```diff
@@ -1,7 +1,42 @@
 import React from "react";
 import InstitutionNavigation from "../Navigation/InstitutionNavigation";
+import { createLecturer,deleteLecturer} from "../../graphql/mutations";
+import { useState } from "react";
+import  {API} from 'aws-amplify';
 
 const AddLecturer = () => {
+
+    const [firstName,setFirstName]=useState("")
+    const [lastName,setLastName]=useState("")
+    const [email,setEmail]= useState("")
+    const [moduleCode,setModuleCode]=useState("")   
+    const add=  async(event) => { 
+        event.preventDefault()
+        console.log("Add lecturer mutation")
+        let lecturer={ 
+            institutionId:"UP123456789",
```

Comment:
Here is my suggestion. When a an institution is created we force its id to be the admins Cognito username.This will allow any admin to easily access their institution.I have implemented this and it seems to work. Furthermore its simple and doesnt require us to change much

Reply:
That should be fine for now I guess but this is not safe security wise and it enforces an unwanted dependancy:

If and admin is fired and a new one created for the institution, then  we have to change the institutionID?
It beats the purpose of having unique Ids since that is how we want to differentiate everything.

But most importantly, the admin has the highest privileges out of all of our users, using their username to mark other objects is not a good idea.